src/core/css/SearchForm.scss

```diff
@@ -2,20 +2,32 @@
 
 .AdminSearchForm-form,
 .SearchForm-form {
+  background: rgba(255, 255, 255, 0.2);
+  border: 0;
   display: flex;
 
-  input[type="search"] {
-    box-sizing: border-box;
-    font-size: $font-size-l;
+  .SearchForm-query {
+    background: url('~core/img/icons/magnifying-glass.svg') 18px 50% no-repeat;
+    background-size: 26px;
+    border: 0;
+    color: $header-font-color;
+    font-size: $font-size-m;
     flex-grow: 1;
     height: 3rem;
-    margin: 0;
-    max-width: 640px;
-    padding: 0.15em 0.25em 0;
-  }
+    margin: 0 auto;
+    max-width: 320px;
+    opacity: 0.75;
+    padding: 0 8px 0 56px;
+    text-align: left;
+    transition: opacity 50ms linear;
+    width: 100%;
 
-  .button {
-    height: 3rem;
-    padding: 0 1rem;
+    &:focus {
```

Comment:
Nice touch. 

I think we'll need an even stronger indicator of focus for accessibility reasons. The goal would be to make sure a keyboard user can see what's focused really clearly if they tab through the controls on the page. Can file a separate issue for that if dealing with it later is better (also could ask for UX input as needed).